Dockerfile.server

```diff
@@ -1,9 +1,21 @@
 # https://versatile.nl/blog/deploying-lerna-web-apps-with-docker
 
-FROM node:16
+FROM buildpack-deps:focal
```

Comment:
Now that we use node:16, doesn't it have the dependencies that you need? If it doesn't can't we just manually install them on top of node:16? I'd rather use an image that specifically for Node if possible.

Reply:
Hmm .. 16 is still based on buster .. but there is 16-bullseye .. so let check if that would do it ..